docs/examples/national_scale/index.md

```diff
@@ -25,6 +32,29 @@ It does not contain much data, but the scaffolding with which to construct and r
 
 ## Model definition
 
+### Referencing tabular data
+
+As of Calliope v0.7.0 it is possible to load tabular data completely separately from the YAML model definition.
+To do this we reference data tables under the `data_tables` key:
+
+```yaml
+--8<-- "src/calliope/example_models/national_scale/model.yaml:data-tables"
+```
+
+In the Calliope national scale example model, we load both timeseries and cost data from file.
+As an example, the data in the cost CSV file looks like this:
+
+{{ read_csv(page.meta.costs.file, header=page.meta.costs.header, index_col=page.meta.costs.index_col) }}
```

Comment:
If I look at the 'built' markdown file here on GitHub, the links to files and to the header do not work properly, but I assume this is because these files are built into documentation by Calliope in a different way than simple markdown visualisation. So perhaps someone else who knows how to build the docs locally can double-check that all is as expected